### ISSUE TITLE
ca-certificates, openssl - something wrong with dependencies

### ISSUE THREAD

**Issue by unspecd** [void-linux/void-packages#29301]
### System

* xuname:  
  *Void 5.10.20_2 x86_64-musl GenuineIntel uptodate rrrnFFFFFFFF*
* package:  
  *ca-certificates-20210119_2*, *openssl-1.1.1j_2*

### Expected behavior
```sh
xbps-remove -Ro # should do nothing
```

### Actual behavior
```
# xbps-remove -Ro
Name    Action    Version           New version            Download size
openssl remove    1.1.1j_2          -                      - 

Size freed on disk:           1718KB
Space available on disk:        14GB

Do you want to continue? [Y/n] 
```

### Steps to reproduce the behavior
```
# xbps-install -Syu || xbps-install -yu xbps
# xbps-install -yu
# xbps-remove -yRo
# xbps-pkgdb ca-certificates
ERROR: ca-certificates: dependency not satisfied: openssl<=2.0_1
```

### Also
```
$ xbps-query -x ca-certificates 
openssl<=2.0_1
run-parts>=0
$ xbps-query -X openssl
$ xbps-query -RX openssl
…
ca-certificates-20210119_2
…
```

**Comment by ericonr** [comment]
Can you show the `xbps-remove -vd -Ro` output? I can't reproduce this locally :/

**Comment by unspecd** [comment]
> I can't reproduce this locally :/

 It does reproduce with the latest [live image](https://alpha.de.repo.voidlinux.org/live/current/void-live-x86_64-20210218.iso).

> Can you show the xbps-remove -vd -Ro output?

https://privatebin.net/?cfce7a0c584a8766#EJWGhK65MsVCdAE7PSfs51UtYepeMjy24DXiic4JYQJS

**Comment by ericonr** [comment]
@Duncaen or @Chocimier any idea? This looks more like a XBPS corner case...

**Comment by unspecd** [comment]
> It does reproduce with the latest live image.

```
$ xuname
Void 5.10.17_1 x86_64 GenuineIntel/KVM uptodate rFF
```

https://privatebin.net/?55c03bd8de5aac83#A2Ai48aSArXHBc2kt6VotzbMHczFp4by2w6sRS2Re6Pe

**Comment by unspecd** [comment]
@ericonr
Please update/replace gist.

**Comment by unspecd** [comment]
grep libressl log.txt
```
[DEBUG] /usr/share/xbps.d/void-virtualpkgs.conf: added virtualpkg openssl for libressl-openssl
[DEBUG] vpkg_user_conf: vpkg_conf openssl pkg libressl-openssl vpkgname openssl
[DEBUG] vpkg_user_conf: vpkg_conf openssl pkg libressl-openssl vpkgname openssl
```

**Comment by Duncaen** [comment]
`/usr/share/xbps.d/void-virtualpkgs.conf` shouldn't exist.

**Comment by unspecd** [comment]
Fixed by 0700dc22f9b23327b6b1f7ec262fbde58b246fc6
